packages/react/src/components/password-input/password-input.tsx

```diff
@@ -0,0 +1,152 @@
+import { ChangeEvent, useState, VoidFunctionComponent, useMemo, FocusEvent } from 'react';
+import styled, { css } from 'styled-components';
+import { FieldContainer } from '../field-container/field-container';
+import { IconButton } from '../buttons/icon-button';
+import { TextInput } from '../text-input/text-input';
+import { useTranslation } from '../../i18n/use-translation';
+import { v4 as uuid } from '../../utils/uuid';
+import { useDataAttributes } from '../../hooks/use-data-attributes';
+import { Tooltip } from '../tooltip/tooltip';
+
+const PasswordContainer = styled.div`
+    border-radius: 0 var(--border-radius) var(--border-radius) 0;
+    display: flex;
+    flex-direction: row;
+    margin-bottom: calc(var(--spacing-1x) * 1.5);
+
+    &:focus-within {
+        border-radius: var(--border-radius);
+
+        /* TODO change when updating thematization */
+        box-shadow: 0 0 0 2px #84c6ea;
+        outline: none;
+
+        input,
+        div + span > button {
+            /* TODO change when updating thematization */
+            border-color: #006296;
+        }
+    }
+`;
+
+const StyledTextInput = styled(TextInput)`
+    flex: 1;
+    margin-bottom: 0;
+
+    input {
+        &::-ms-reveal {
+            display: none;
+        }
+
+        border-radius: var(--border-radius) 0 0 var(--border-radius);
+        border-width: 1px 0 1px 1px;
+    }
+`;
+
+const StyledIconButton = styled(IconButton) <{ isValid: boolean }>`
+    background-color: white;
```

Comment:
Peut-être utiliser le thème
```suggestion
    background-color: ${({ theme }) => theme.greys.white};
```